frontend/src/components/inputs/Selector.vue

```diff
@@ -39,7 +39,8 @@ import vSelect from 'vue-select';
 import type { SelectorOption } from '@/types/form';
 
 interface SelectorContext extends FormKitFrameworkContext {
-  options: Array<SelectorOption>;
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
+  options: Array<SelectorOption<any>>;
```

Comment:
Can it be `undefined` or is `unknown` sufficient? 🤔

Reply:
Good point. `unknown` is better here

Reply:
So we safe the `eslint-disable` comment. 🎉